hub/util/tests/test_slice.py

```diff
@@ -0,0 +1,46 @@
+from hub.util.slice import merge_slices
+
+
+RANGE = range(100)
+
+
+def test_merge_slices_first_longer():
```

Comment:
These tests can be changed into one test that takes in a list of inputs and expected outputs and runs the test on the entire list of testcases.

Reply:
This is what I started with initially.
I found that it was difficult to separate the cases conceptually. When a case fails, it's hard for me to tell what actually went wrong when I can't check the case title.

Is there a good way to annotate this in pytest? I will investigate

Reply:
I found pytest-cases, which is good enough. Updated.